rosdep/base.yaml

```
@@ -6466,6 +6466,17 @@ log4cxx:
   slackware: [apache-log4cxx]
   ubuntu:
     '*': [liblog4cxx-dev]
+lsb-release:
+  arch: [lsb-release]
+  debian: [lsb-release]
+  fedora: [lsb_release]
+  gentoo: [sys-apps/lsb-release]
+  nixos: [lsb-release]
+  openembedded: [lsb-release@openembedded-core]
+  opensuse: [lsb-release]
+  rhel: [lsb-release]
```

According to `dnf whatprovides lsb_release` on Alma 8 the package name is `redhat-lsb-core` there.

```suggestion
  rhel:
    '*': [lsb-release]
    '8': [redhat-lsb-core]
```

---

This gets into the weeds a little, but it would also be possible to specify _the thing you actually want_ in this rosdep rule like this:
```suggestion
  rhel: [/usr/bin/lsb_release]
```

On RHEL 8, it appears to be provided by `redhat-lsb-core` and on RHEL 9, it's provided by `lsb_release` from EPEL. The EPEL package on RHEL 9 also provides a virtual package called `lsb-release`, which is why the current rule works for RHEL 9.

---

> The EPEL package on RHEL 9 also provides a virtual package called `lsb-release`,

Yep, I sow the EPEL entry in https://repology.org/project/lsb-release/versions and also that if matches Fedora and I wrongly assumed that will work for all variants. Thanks for the hints.

---

Ah crud, this was my mistake. Evidently I taught rosdep how to deal with virtual packages, but not files. Steven!'s suggestion would be best.